2-TokenCache/Console-TokenCache/Program.cs

```diff
@@ -0,0 +1,218 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Graph;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.Extensions.Msal;
+using System;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace Console_TokenCache
+{
+    class Program
+    {
+        private static PublicClientApplicationOptions appConfiguration = null;
+        private static IConfiguration configuration;
+        private static string _authority;
+
+        static async Task Main(string[] args)
```

Comment:
This method is too long - please separate out a few private methods.

Reply:
For the sake of simplicity and readability we often have to compromise on a few coding practices like small methods , OOPS principals and such. For educational purpose, we got some feedback that having less "production like code" is easier for developers to understand the sample, and for us to document it. 
I have discussed this with Kalyan, and we agreed to keep `Main()` as is.

Reply:
As you wish. I wasn't talking about design patterns or anything. Main is a really long method and it could easily be broken down into: 

- setting up the MsalCacheHelper
- acquiring tokens
- calling protected resource